src/model/Model.cpp

```diff
@@ -1,12 +1,11 @@
 #include "Model.h"
 #include <iostream>
 
-Model::Model(string name) {
-    tableName = name;
-}
+Model::Model(string name) :
+    tableName(name) {}
 
 Model::~Model() {
-    for (vector<Field*>::const_iterator it = fields.begin(); it != fields.end(); it++) {
+    for (vector<Field*>::const_iterator it = fields.begin(); it != fields.end(); ++it) {
```

Comment:
This likely to be a breaking change as it changes the logic. Putting the ++ before `it` means that it increments before going into the loop, thereby skipping the first `Field` in the vector. It would also likely be a very difficult bug to find in the future.